.github/workflows/package-publish.yml

```diff
@@ -2,14 +2,35 @@ name: Publish package to PyPI and TestPyPI
 
 on:
   push:
-    branches: [ main ]
     tags: [ '[0-9]+.[0-9]+.[0-9]+' ]
-
 jobs:
+  # @see https://stackoverflow.com/a/72959712/8179249
```

Comment:
minor: I'm reluctant to leave links and commented code in the source tree. I would add it to the description of the associated ticket (if any) (unless it's a temporary change)

Reply:
I concur, will resolve